kong/tools/queue.lua

```diff
@@ -94,9 +94,17 @@ local Queue_mt = {
   __index = Queue
 }
 
+-- string buffer for constructing queue keys efficiently
+local buf = require("string.buffer").new(32)
+
 
 local function _make_queue_key(name)
-  return (workspaces.get_workspace_id() or "") .. "." .. name
+  local key = buf:put(
```

Comment:
Do we have evidence that for this short key `string.buffer` is faster than concat?

Reply:
No, the buffer is much slower, reverted.